docs/api/classifier/rocauc.rst

```diff
@@ -3,44 +3,92 @@
 ROCAUC
 ======
 
-A ROCAUC (Receiver Operating Characteristic/Area Under the Curve) plot allows the user to visualize the tradeoff between the classifier's
-sensitivity and specificity.
+A ``ROCAUC`` (Receiver Operating Characteristic/Area Under the Curve) plot allows the user to visualize the tradeoff between the classifier's sensitivity and specificity.
+
+The Receiver Operating Characteristic (ROC) is a measure of a classifier's predictive quality that compares and visualizes the tradeoff between the model's sensitivity and specificity. When plotted, a ROC curve displays the true positive rate on the Y axis and the false positive rate on the X axis on both a global average and per-class basis. The ideal point is therefore the top-left corner of the plot: false positives are zero and true positives are one.
+
+This leads to another metric, area under the curve (AUC), which is a computation of the relationship between false positives and true positives. The higher the AUC, the better the model generally is. However, it is also important to inspect the "steepness" of the curve, as this describes the maximization of the true positive rate while minimizing the false positive rate.
 
 .. code:: python
 
     from sklearn.model_selection import train_test_split
 
     # Load the classification data set
-    data = load_data("occupancy")
+    data = load_occupancy()
 
     # Specify the features of interest and the classes of the target
     features = ["temperature", "relative humidity", "light", "C02", "humidity"]
     classes = ["unoccupied", "occupied"]
 
     # Extract the numpy arrays from the data frame
-    X = data[features].as_matrix()
-    y = data.occupancy.as_matrix()
+    X = data[features].values
+    y = data.occupancy.values
 
     # Create the train and test data
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
```

Comment:
Since `load_occupancy()` contains time series data, should we use `sklearn.model_selection.TimeSeriesSplit` instead of `sklearn.model_selection.train_test_split`. `train_test_split` assumes the observations are independent and shuffles the data points by default when making the split. However, this wouldn't be true for time series datasets such as `occupancy` (which includes per-minute observations from February 4, 2015 through February 18, 2015). Here is a short excerpt from `TimeSeriesSplit`'s [documentation](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.TimeSeriesSplit.html):

> Time Series cross-validator
> 
> Provides train/test indices to split time series data samples that are observed at fixed time intervals, in train/test sets. In each split, test indices must be higher than before, and thus shuffling in cross validator is inappropriate.

Reply:
This is fascinating. Since it seems to apply more broadly to our use of the occupancy dataset throughout our docs, though, perhaps it is better suited to a new issue? Want to open one?

Reply:
Sure @rebeccabilbro! We do use the `occupancy` dataset in a lot of our examples, so that's a good idea.